misc/scripts/install/docker-volume-vsphere.conf

```diff
@@ -7,4 +7,4 @@ respawn
 
 kill timeout 20
 
-exec /usr/local/bin/$UPSTART_JOB
+exec /usr/local/bin/$UPSTART_JOB -port 1019
```

Comment:
(not an issue, just a question) - do we need to pass port number ? Can the  default be hardcoded in client server